### PR TITLE
To Do List Add-Remove-Edit

### PR DESCRIPTION
**Pull request by Johnadibe**
- Used webpack to bundle JavaScript.
- Used ES6 module to write modular Javascript
- Implemented a function for adding a new task to an array.
- Implemented a function for deleting a task from an array.
- Implemented a function for editing task descriptions.
- Saved all To-Do List changes in local storage.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,43 +1,69 @@
...
-    description.innerHTML = listData[i].description;
-    listItem.appendChild(description);
+const deleteTodo = (index) => {
+  const newTodo = todos.filter((element) => element.index !== index);
+  todos.length = 0;
+  let i = 0;
+  newTodo.forEach((element) => {
+    element.index = i;
+    i += 1;
+  });
```
Comment (roshan-bajgain) [inline src/index.js]: - [ ] According to the project requirement, the property index is set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal 5). Your index value is starting from 0. please make it 1.

File: README.md [hunk 2]
```diff
@@ -2,13 +2,14 @@
...
-This project is to build a simple Html list of To Do tasks.
+This project is to implement the CRUD (create, read, update, delete) methods in a simple list of To Do tasks.
 
 ## Built With
 
 - HTML
 - CSS
 - JavaScript
+- Webpack
 - Visual Studio Code
```
Comment (roshan-bajgain) [inline README.md:13]: - [ ] Please write how to install the project and how to start the project in the `Getting Started` section of your README file. ie:
`npm install` to install the dependence
`npm start` to run the project

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @Johnadibe, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Johnadibe 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - No linter errors. ✔️
 - GitHub flow is followed. ✔️
 - PR has a good title and summary. ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**